CHANGELOG.md

```diff
@@ -1,3 +1,69 @@
+7.0.0
+-------------------
+ * Require Node.js 18 or newer
```

Comment:
I assume the CLI is only released with the next major release? Otherwise, some builds will fail after the package update, although the rest of the SDK is 16+

Reply:
v7 is slated for release Friday May 10, 2024. I've published 9 beta releases and 2 release candidates. I have resolved all issues. Nobody has reported a failed build using RC2. The CLI requires Node 18 because we don't have the resources to support Node 16. The next major release of the SDK can bump the required Node version to 18, but we don't have to until we update the dependencies in the SDK. This may not happen until 2 major releases from now, but by then we'll drop Node 18 too.